specification/network/resource-manager/readme.md

```diff
@@ -31,6 +31,76 @@ openapi-type: arm
 tag: package-2023-11
 ```
 
+### Tag: package-2023-11-preview
+These settings apply only when `--tag=package-2023-11-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-11-preview'
+input-file:
+  - Microsoft.Network/preview/2022-02-01-preview/networkManager.json
```

Comment:
Can you double check there are no major changes to network manager since this spec? If we have deprecated fields included in the spec, your network manager CRUD commands could all result in 500's or unexpected backend behavior. Ideally if the CRUD no longer works we would have retired the API version, but we should still check.

Reply:
I will add the changes in networkmanager.json via autorest instead